contracts/characters.sol

```diff
@@ -26,7 +26,7 @@ contract Characters is Initializable, ERC721Upgradeable, AccessControlUpgradeabl
     }
 
     function migrateTo_1ee400a() public {
-        require(hasRole(DEFAULT_ADMIN_ROLE, msg.sender), "Not admin");
+        require(hasRole(DEFAULT_ADMIN_ROLE, msg.sender), "NA");
```

Comment:
would recommend removing the "NA" strings entirely from all migration functions. Saves a decent bit on contract size